### ISSUE TITLE
 Error after upgrade to Rails 6.0.0

### ISSUE THREAD

**Issue by tiagocassio** [lynndylanhurley/devise_token_auth#1329]
I've update Rails to 6.0.0 final version and after call "rails s", i'm getting this error:

`expected file /home/cassio/.rvm/gems/ruby-2.6.3/bundler/gems/devise_token_auth-2c3a8d833040/app/models/devise_token_auth/concerns/tokens_serialization.rb to define constant DeviseTokenAuth::Concerns::TokensSerialization, but didn't (NameError)`

gem version: 1.1.1 (branch master)
devise version: 4.6.2 (branch master)
ruby version: 2.6.3

**Comment by jenkijo** [comment]
I can't install devise-token-auth with Rails 6.0.0

```
gem 'devise_token_auth', github: 'lynndylanhurley/devise_token_auth'
```

```
Bundler could not find compatible versions for gem "railties":
  In snapshot (Gemfile.lock):
    railties (= 6.0.0)

  In Gemfile:
    devise_token_auth was resolved to 1.1.1, which depends on
      devise (> 3.5.2, < 4.7) was resolved to 3.5.10, which depends on
        railties (>= 3.2.6, < 5)

    rails (~> 6.0) was resolved to 6.0.0, which depends on
      railties (= 6.0.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by tadaniya** [comment]
I hit the same problem...
I downgrade devise-token-auth to 1.1.0

**Comment by BilalBudhani** [comment]
I'm facing the same issue. I tried downgrading to `1.1.0` as suggested by @tadaniya but bundler throws dependencies mismatch error.

**Comment by ponyczek** [comment]
I am getting the same error.

**Comment by ekurennyy** [comment]
Hello guys.
That is what I had today:

Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 6.0.0)

  In Gemfile:
    rails (~> 6.0.0)

    devise_token_auth was resolved to 0.1.21.alpha2, which depends on
      rails (~> 4.1.4)

Is this gem should work with Rails 6.0.0 now?..

**Comment by BilalBudhani** [comment]
I've patched the issue in my personal fork https://github.com/BilalBudhani/devise_token_auth/tree/token-serialization-fix for now. I'll find some time in coming weeks to go through contribution guideline of this repo & raise PR.
